Simple Featuretools RUL Demo.ipynb

```diff
@@ -1164,21 +1263,14 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
+   "execution_count": 16,
    "metadata": {},
    "outputs": [],
    "source": [
     "# Save output files\n",
-    "\n",
-    "import os\n",
-    "\n",
-    "try:\n",
-    "    os.mkdir(\"output\")\n",
-    "except:\n",
-    "    pass\n",
-    "\n",
+    "os.makedirs('output', exist_ok=True)\n",
```

Comment:
This only works in Python 3.  Does this demo require python 3?

Reply:
Yes, support for Compose starts on 3.5.